raiden/network/proxies/token_network.py

```diff
@@ -1011,8 +1011,7 @@ def _update_preconditions(
             #
             # Exception is raised if the public key recovery failed.
         except Exception:  # pylint: disable=broad-except
-            msg = "Couldn't verify the balance proof signature"
-            return RaidenUnrecoverableError, msg
```

Comment:
Ah damn ... that would have been caught by mypy so easily ....

How come this was not caught by a test? Can you maybe add one?

Reply:
I added one in this PR myself: https://github.com/raiden-network/raiden/pull/3331/commits/0b9a02f2ea456e8b7026dbaed9f6ba587086d2e9